waveform_editor/shape_editor/plasma_shape.py

```diff
@@ -0,0 +1,184 @@
+import math
+
+import holoviews as hv
+import imas
+import numpy as np
+import panel as pn
+import param
+from panel.viewable import Viewer
+
+from waveform_editor.util import EquilibriumInput
+
+
+class PlasmaShapeParams(param.Parameterized):
```

Comment:
Note: current widgets are probably too course, it's only possible to adjust in increments of 0.1, while more finegrained control will probably be necessary.